lexicon/conversion.py

```diff
@@ -348,7 +349,10 @@ def __init__(
         self.set_vis_name("Convert Between Regional Orth Spellings")
 
         self.bliss_lexicon = bliss_lexicon
-        self.orth_mapping_file = orth_mapping_file
```

Comment:
This would incur in the same issue as you're seeing right now, because this is done by the manager. This if should be done on the `run()` function instead.